Update DDoS option for secure hub.

cgoenka1 opened the pull request:
With the Customer Provided IP option, DDoS should be enabled for secure hub.

cgoenka1 committed:
Update vhubs-and-vnets.md

cgoenka1 committed:
Update vhubs-and-vnets.md

duongau commented on articles/firewall-manager/vhubs-and-vnets.md:
```diff
@@ -39,7 +39,7 @@ The following table compares these two architecture options and can help you dec
 |**Web Application Firewall on Application Gateway** |Supported in Virtual Network|Currently supported in spoke network|
 |**Network Virtual Appliance**|Supported in Virtual Network|Currently supported in spoke network|
-|**Azure DDoS Protection support**|Yes|No|
+|**Azure DDoS Protection support**|Yes|Yes (With Customer Provided IP)|
```
```suggestion
|**Azure DDoS Protection support**|Yes|Yes ([With customer provided public IP](../firewall/secured-hub-customer-public-ip.md))|
```

ktoliver committed:
Clarify Azure DDoS Protection support details

Updated the description for Azure DDoS Protection support to include details about customer-provided public IP address.
